packages/shared/src/lib/featureManagement.ts

```diff
@@ -48,4 +48,6 @@ const feature = {
   }),
 };
 
+export const featureAutorotateAds = new Feature('autorotate_ads', 1);
```

Comment:
Default should be null/undefined right? It should be turned off by default

Reply:
```suggestion
export const featureAutorotateAds = new Feature('autorotate_ads', 0);
```

Reply:
Default is 0 supposed to be 0

Reply:
You control the duration of how long to render each ad (in ms)